torchaudio/functional.py

```diff
@@ -313,6 +313,13 @@ def create_fb_matrix(
         enorm = 2.0 / (f_pts[2:n_mels + 2] - f_pts[:n_mels])
         fb *= enorm.unsqueeze(0)
 
+    if (fb.max(dim=0).values == 0.).any():
+        warnings.warn(
+            "At least one mel filterbank has all zero values."
```

Comment:
I think you need a space after the period, otherwise the constructed string would not have a space between two sentences.

```suggestion
            "At least one mel filterbank has all zero values. "
```